### ISSUE TITLE
About Imagenet data classification test

### ISSUE THREAD

**Issue by 15122901359** [Res2Net/Res2Net-PretrainedModels#25]
@gasvn 
Hello, I used the model you provided and the model transformed into gloun to test and verify the classification results of ImageNet. The model test results of the provided pytorch version are very low in accuracy of about 20%, but the converted gloun model can achieve about 71% (the two data operations are consistent). So I would like to ask you, do you need to do anything special before data entry?
The following is my data processing of the two. Could you please help me to see what the problem is? Thank you very much.
pytorch data processing
```
   class TestDataset(torch.utils.data.Dataset):
    def __init__(self, file, root, **kwargs):
        # mean and std
        self._root = os.path.expanduser(root)
        self.labelfile = file
        self._exts = ['.jpg', '.jpeg', '.png']
        self.normalize = transforms.Normalize(
            mean=[0.485, 0.456, 0.406],
            std=[0.229, 0.224, 0.225])
        self.size = (224,224)
        self.parse_input_list(self._root,self.labelfile)
        self.num_sample = len(self.list_sample)
        self.tranform = transforms.Compose([
            transforms.Resize(256),
            transforms.CenterCrop(224),
            transforms.ToTensor(),
            self.normalize,
        ])

    def parse_input_list(self, path,labelfile):
        self.list_sample = []
        labellist = {}
        with open(labelfile, "r") as f:  # 打开文件
            data = f.read().split('\n')[:-1]
            for i in range(len(data)): labellist[data[i].split(' ')[0]] = data[i].split(' ')[1]
            for filename in sorted(os.listdir(path)):
                if filename in labellist.keys():
                    label = labellist[filename]
                filename = os.path.join(path, filename)
                ext = os.path.splitext(filename)[1]
                if ext.lower() not in self._exts: print(
                    'Ignoring %s of type %s. Only support %s' % (filename, ext, ', '.join(self._exts)))
                self.list_sample.append((filename, label))
    def img_transform(self,img):
        img = self.tranform(img)
        return img.cuda(0)

    def __getitem__(self, index):
        from PIL import Image
        image_path,label = self.list_sample[index]
        img = Image.open(image_path).convert('RGB')
        if self.img_transform:
            img = self.img_transform(img)
        return img,label

    def __len__(self):
        return self.num_sample
 ```
gloun data processing
 ```
class TestDataset(gdata.Dataset):
    def __init__(self, root, labelfile, flag=1, transform=None):
        self._root = os.path.expanduser(root)
        self._flag = flag
        self._transform = transform
        self._exts = ['.jpg', '.jpeg', '.png']
        self.labelfile = labelfile
        self._list_images(self._root,self.labelfile)
    def _list_images(self, path,labelfile):
        self.items = []
        labellist = {}
        with open(labelfile, "r") as f:  # 打开文件
            data = f.read().split('\n')[:-1]
            for i in range(len(data)):labellist[data[i].split(' ')[0]] = data[i].split(' ')[1]
            for filename in sorted(os.listdir(path)):
                if filename in labellist.keys():
                    label = labellist[filename]
                filename = os.path.join(path, filename)
                ext = os.path.splitext(filename)[1]
                if ext.lower() not in self._exts: print('Ignoring %s of type %s. Only support %s' % (filename, ext, ', '.join(self._exts)))
                self.items.append((filename, label))

    def __getitem__(self, idx):

        img = image.imread(self.items[idx][0], self._flag)
        label = self.items[idx][1]
        if self._transform is not None:
            return self._transform(img, label)
        return img, label

    def __len__(self):
        return len(self.items)
def transform_train(data, label):
    im1 = image.imresize(data.astype('float32') / 255, 224, 224)
    auglist1 = image.CreateAugmenter(data_shape=(3, 224, 224), resize=0,
                        rand_crop=False, rand_resize=False, rand_mirror=False,
                        mean=numpy.array([0.485, 0.456, 0.406]), std=numpy.array([0.229, 0.224, 0.225]),
                        brightness=0, contrast=0,
                        saturation=0, hue=0,
                        pca_noise=0, rand_gray=0, inter_method=2)

    for aug in auglist1:
        im1 = aug(im1)
    # 将数据格式从"高*宽*通道"改为"通道*高*宽"。
    im1 = nd.transpose(im1, (2,0,1))
    # im1 = nd.expand_dims(data=im1, axis=0)
    return (im1, nd.array([label],ctx=mx.gpu()).asscalar().astype('float32'))
 ```
